keypoints/include/pcl/keypoints/impl/iss_3d.hpp

```diff
@@ -288,7 +288,7 @@ pcl::ISSKeypoint3D<PointInT, PointOutT, NormalT>::initCompute ()
       return (false);
     }
   }
-  else if (border_radius_ < 0.0)
+  else
```

Comment:
What happens if the border_radius_ is 0 ?

Reply:
I guess before that case wasn't properly handled, since it was either >0 or <0 but not equal to 0. Now it will end in the else case and return false.

Reply:
Oh, actually it seems that setting `border_radius_` to 0 is a valid choice if border estimation is not desired. So I reverted this change.